parallel.yaml

```diff
@@ -35,7 +41,20 @@ pipeline:
       ${{targets.contextdir}}/usr/bin/env_parallel.ksh \
       ${{targets.contextdir}}/usr/bin/env_parallel.mksh \
       ${{targets.contextdir}}/usr/bin/env_parallel.pdksh \
-      ${{targets.contextdir}}/usr/bin/env_parallel.tcsh
+      ${{targets.contextdir}}/usr/bin/env_parallel.tcsh \
+      ${{targets.contextdir}}/usr/bin/env_parallel.dash \
+      ${{targets.contextdir}}/usr/bin/env_parallel.fish \
+      ${{targets.contextdir}}/usr/bin/env_parallel.sh \
+      ${{targets.contextdir}}/usr/bin/env_parallel.zsh
```

Comment:
Since these will no longer result in a large dependency tree, can we leave them?

Reply:
I think we should continue to omit the `env_parallel.*` for shells that we do not ship. But I think it would be good to continue shipping the scripts for the shells that we *do* ship. In which case, the number of provides should remain unchanged from the current version.

Reply:
For the sake of completeness, I'll note that one option here is to move all of these snippets into individual subpackages - `parallel-shell-zsh`, `parallel-shell-mksh`, etc, each depending on the corresponding shell binary package. Each of these could provide `parallel-shell`, which `parallel` itself could depend on. That feels like overkill for this case, but worth explicitly mentioning even if we agree it's too much.

Reply:
I have added it again for `fish`, `zsh` and `sh` shell. On adding separate compat subpackages, we can do that if we require it sometimes in future, for now I think we're not using them anywhere.

Reply:
I see `dash` continues to be omitted, why is that?

Reply:
missed it, now we are shipping env_parallel.dash too.

Reply:
thanks!